webrender/res/ps_border_corner.vs.glsl

```diff
@@ -189,6 +189,10 @@ void main(void) {
             vAlphaSelect = 1.0;
             vSDFSelect = 1.0;
             break;
+        case BORDER_STYLE_DOTTED:
+            // Disable normal clip radii for dotted corners, since
+            // all the clipping is handled by the clip mask.
+            vClipSign = vec2(0.0);
```

Comment:
I recall some Android devices no appreciating `case` fall-through. Let's have a `break` just to be safe